### DISCUSSION TITLE
Mouse Axis to keyboard

### DISCUSSION THREAD

**Discussion by thedrakeraider** [sezanzeb/input-remapper#456] [Q&A]
I saw a question like this earlier, but I would like to do the opposite. I'm trying to use my mouse axis to control my arrow keys on my keyboard. So when I move the mouse, it enters the left, right, up and down keys as long as I'm moving in that direction. Any advice?

**Comment by jonasBoss** [comment]
this is possible on the beta branch. There you can map the mouse axis as a key or macro. Be aware that this will disable the mouse axis! 
You can also map any mouse button + axis as a key, that way the mouse axis is only disabled when you hold that key.

**Reply by Daronfritts** [reply]
How do you do this. I’m trying to use a modifier with the arrow keys to move mouse axis

**Reply by jonasBoss** [reply]
what you want seems to be Keyboard -> Mouse and not Mouse -> Keyboard.

See [mouse macro](https://github.com/sezanzeb/input-remapper/blob/main/readme/macros.md#mouse) for that
